external/model-preparation-algorithm/mpa_tasks/apis/classification/task.py

```diff
@@ -229,17 +231,34 @@ def train(self,
               output_model: ModelEntity,
               train_parameters: Optional[TrainParameters] = None):
         logger.info('train()')
+        # Check for stop signal between pre-eval and training. 
+        # If training is cancelled at this point,
+        if self._should_stop:
```

Comment:
What do you think of renaming from '_should_stop' to '_is_cancelled'?

Reply:
The variable '_should_stop' is a common variable used in other tasks(mmdet/mmseg/deep-reid). So it doesn't seem appropriate to change.

Reply:
I see. If the name is common one being used for the other backends, yes then it should be kept.